### DISCUSSION TITLE
can't close the connection when the code ended running

### DISCUSSION THREAD

**Discussion by atef7534** [explosion/spaCy#11795] [Help: Coding & Implementations]
the connection is still suspended in jupyter notebook and I can't finish it to do another operation 
and another problem is when using displacy.serve or render it doesn't show me the result but the code is done it sends me to go to http://0.0.0.0:5000 and when I open it it said that 
The webpage at http://0.0.0.0:5000/ might be temporarily down or it may have moved permanently to a new web address.
ERR_ADDRESS_INVALID

**Comment by polm** [comment]
What do you mean by "the connection is still suspended"? Do you mean you can't push the run button again? 

Can you provide an example of the code that caused this? Without any code it's hard to figure out what's going on.

> The webpage at http://0.0.0.0:5000/ might be temporarily down or it may have moved permanently to a new web address.
ERR_ADDRESS_INVALID

This is just the normal error if the server isn't running.
